source_test.yaml

```diff
@@ -176,6 +176,20 @@
   editable: False
   strict_validation: True
 
+- name: 'drupal'
+  versions_from_repo: False
+  type: 0
+  ignore_patterns: ['^(?!DRUPAL-).*$']
+  directory_path: 'advisories'
+  repo_url: 'https://github.com/ackama/drupal-advisory-database.git'
```

Comment:
Before this gets merged, it would be good to discuss the location of the repo_url, whether it should be on drupal.org or github, if on github where. Thanks!

I'll reach out to discuss.